integration/setup/config.go

```diff
@@ -30,25 +31,23 @@ func NewConfig() (Config, error) {
 		}
 	}
 
-	var cpK8sClients *k8sclient.Clients
-	{
-		c := k8sclient.ClientsConfig{
-			Logger: logger,
-
-			KubeConfigPath: env.KubeConfigPath(),
-		}
+	config, err := clientcmd.BuildConfigFromFlags("", env.KubeConfigPath())
```

Comment:
Found this function in client-go tutorial page; https://github.com/kubernetes/client-go/blob/master/examples/out-of-cluster-client-configuration/main.go#L58-L62